std/process.d

```diff
@@ -296,8 +308,9 @@ static:
     multi-threaded programs. See e.g.
     $(LINK2 https://www.gnu.org/software/libc/manual/html_node/Environment-Access.html#Environment-Access, glibc).
     */
-    void remove(scope const(char)[] name) @trusted nothrow @nogc
+    void remove(scope const(char)[] name) @trusted
```

Comment:
I'm not sure we can remove those attributes, this might break code.

Why does this now require GC and throwing? Can we catch any exceptions?

Reply:
Also, the comment for this function should be altered, as now we are supporting multithreaded changes via our own lock.